studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyRow.tsx

```diff
@@ -34,8 +34,8 @@ const PolicyRow: FC<Props> = ({
         </div>
         <div className="flex items-center space-x-2">
           <p className="text-scale-1000 text-sm">Applied to:</p>
-          {policy.roles.map((role) => (
-            <code className="text-scale-1000 text-xs">{role}</code>
+          {policy.roles.map((role, i) => (
```

Comment:
@Isaiah-Hamilton any of these prone to rerender and need a more specific key?  `policy-${role}-${i}` or similar?

Reply:
I don't think its necessary to have more specific keys, but can be nice to have.